embedded-graphics/src/lib.rs

```diff
@@ -283,7 +283,7 @@
 //!     style::PrimitiveStyle,
 //! };
 //!
-//! Circle::new(Point::new(32, 32), 20)
+//! Circle::new(Point::new(16, 16), 40)
```

Comment:
Not sure why this changed, but the circle is no longer centered in the screenshot.

Reply:
This problem was caused because the examples weren't updated after the recent circle changes. I just quickly adjusted the values to be closer to the original, but if you want I can also center it perfectly. But a centered circle should probably be constructed with the `with_center` constructor instead.